src/parser/csql_grammar.y

```diff
@@ -4343,7 +4443,37 @@ drop_stmt
 			$$ = node;
 			PARSER_SAVE_ERR_CONTEXT ($$, @$.buffer_pos)
 
-		DBG_PRINT}}                
+		DBG_PRINT}}
+	| DROP				/* 1 */
+		{ push_msg (MSGCAT_SYNTAX_SYNONYM_INVALID_DROP); }	/* 2 */
+	  opt_access_modifier		/* 3 */
+	  SYNONYM			/* 4 */
+	  opt_if_exists			/* 5 */
+	  synonym_name			/* 6 */
+	  	{ pop_msg(); }
+		{{
+
```

Comment:
Please add below
`DBG_TRACE_GRAMMAR(drop_stmt, | DROP SERVER opt_access_modifier SYNONYM opt_if_exists  synonym_name );`

Reply:
Thank you. Added.